src/GROWdb_variable_puller.Rmd

```diff
@@ -28,7 +30,7 @@ sf_use_s2(FALSE)
 # url = paste0("https://edap-ow-data-commons.s3.amazonaws.com/NHDPlusV21/", "Data/NationalData/NHDPlusV21_NationalData_Seamless", "_Geodatabase_Lower48_07.7z"),
```

Comment:
add reproducibility here - either as an if/else (i.e. if this folder exists, just keep going, otherwise, make the folder and download the file.... OR... use NHDPlusTools.

Reply:
There are a few other times where the data folder is referenced, but it doesn't technically exist, so for reproducibility, I'd add a similar if/else there too (looks like the first instance is ~ line 97 for data/mid)